configs/test-utils/src/ThrowError.tsx

```diff
@@ -1,11 +1,21 @@
 import type { PropsWithChildren } from 'react'
-import { useEffect, useState } from 'react'
+import { useEffect, useLayoutEffect, useRef, useState } from 'react'
+
+const isClient = typeof window !== 'undefined'
+export const useIsomorphicLayoutEffect = isClient ? useLayoutEffect : useEffect
+
+export const useTimeout = (fn: () => void, ms: number) => {
+  const fnRef = useRef(fn)
+
+  useIsomorphicLayoutEffect(() => {
```

Comment:
I used useIsomorphicLayoutEffect in this time, thanks to @minsoo-web 

I referred to [this code of usehooks-ts recommended](https://github.com/juliencrn/usehooks-ts/blob/master/packages/usehooks-ts/src/useTimeout/useTimeout.ts) by @ssi02014 a lot. thanks!👍